docgen/webpack.config.babel.js

```diff
@@ -24,6 +25,8 @@ export default {
   resolve: {
     alias: {
       'react-instantsearch': join(__dirname, '../packages/react-instantsearch/'),
+      'material-ui': join(__dirname, '../node_modules/material-ui/'),
```

Comment:
Is this mandatory? I would say there should be no need to do so. We do it for react-instantsearch only to link a local library